drivers/video/Kconfig.mt9m114

```
@@ -6,5 +6,7 @@
 config VIDEO_MT9M114
 	bool "MT9M114 Aptina CMOS digital image sensor"
 	depends on I2C
+	depends on DT_HAS_APTINA_MT9M114_ENABLED
+	default DT_HAS_APTINA_MT9M114_ENABLED
```

What is the difference with or advantage over:
```suggestion
	default y
```

---

@pdgendt there isn't one.  It made a bit more sense before I added the `depends on`.  But with that, it should just be `default y`.